src/openMVG/types.hpp

```diff
@@ -28,19 +28,15 @@ typedef std::pair<IndexT,IndexT> Pair;
 typedef std::set<Pair> Pair_Set;
 typedef std::vector<Pair> Pair_Vec;
 
-#define OPENMVG_NO_UNORDERED_MAP 1
-
-#if defined OPENMVG_NO_UNORDERED_MAP
+#if defined OPENMVG_UNORDERED_MAP
```

Comment:
I think it's just a choice to do regarding which container is the most efficient.
But as the test is not done, we have kept this define.